### ISSUE TITLE
Strict apolloClientVersion value check breaks fragment-matcher

### ISSUE THREAD

**Issue by michalkvasnicak** [dotansimha/graphql-code-generator#3464]
**Describe the bug**

Specifying `apolloClientVersion` as integer for fragment-matcher results in error `Extension .ts not supported` because strict match against string is used at https://github.com/dotansimha/graphql-code-generator/blob/master/packages/plugins/other/fragment-matcher/src/index.ts#L134

In docs it is specified as integer: https://graphql-code-generator.com/docs/plugins/fragment-matcher#apolloclientversion-string-default-value-2

**To Reproduce**

1. create configuration for example

```yaml
overwrite: true
schema: './schema/index.ts'
documents: './graphql/**/*.graphql'
generates:
  './graphql/__generated__/introspectionResult.ts':
    hooks:
      afterOneFileWrite:
        - prettier --write
    config:
      apolloClientVersion: 2
    plugins:
      - fragment-matcher
```

2. try to generate schema, you will get `Extension .ts not supported`.

3. Then change configuration to

```yaml
overwrite: true
schema: './schema/index.ts'
documents: './graphql/**/*.graphql'
generates:
  './graphql/__generated__/introspectionResult.ts':
    hooks:
      afterOneFileWrite:
        - prettier --write
    config:
      apolloClientVersion: '2'
    plugins:
      - fragment-matcher
```

4. try to generate schema, everything is ok.

**Environment:**

- OS: Mac OS 10.15.2 
- `@graphql-codegen/fragment-matcher@1.12.2`: 
- NodeJS: 12.13.0

**Comment by dotansimha** [comment]
You are right, we should allow more loose type checking for that values because it's coming from the YAML config. 

I changed it in: https://github.com/dotansimha/graphql-code-generator/pull/3495 
Now both string/number will work . 

Can you please try the alpha version? `1.12.3-alpha-1823fb22.25`

**Comment by michalkvasnicak** [comment]
@dotansimha I tried the version you mentioned and it works OK. 👍  Thank you. I'm sorry I didn't provide a PR. I was in a hurry :(

**Comment by dotansimha** [comment]
Fixed in `v1.13.0`
